code/go/0chain.net/blobbercore/handler/grpc_handler_integration_test.go

```diff
@@ -1161,37 +1161,49 @@ func TestBlobberGRPCService_IntegrationTest(t *testing.T) {
 		root, _ := os.Getwd()
 		path := strings.Split(root, `code`)
 
+		err := os.MkdirAll(path[0]+`docker.local/blobber1/files/files/exa/mpl/eId/objects/tmp/Mon/Wen`, os.ModePerm)
+
 		f, err := os.Create(path[0] + `docker.local/blobber1/files/files/exa/mpl/eId/objects/tmp/Mon/Wen/MyFile`)
 		if err != nil {
 			t.Fatal(err)
 		}
+		defer f.Close()
+		defer func() {
+			err := os.RemoveAll(path[0] + `docker.local/blobber1/files/files/exa/mpl/eId/objects/tmp/Mon`)
+			if err != nil {
+				t.Fatal(err)
```

Comment:
this should be moved just after mkdir, it wont get removed if create fails

Reply:
Noted